blog/forge-v6-release.md

```diff
@@ -0,0 +1,205 @@
+
+The `plugins` array now takes objects containing an object with properties `name` and `config`, rather than tuples containing the plugin name and config.
+
+This aligns the syntax for this configuration with the `publishers` and `makers` arrays.
+
+```diff
+{
+  plugins: [
+-      [
+-        '@electron-forge/plugin-webpack',
+-        { /* ... */ }
+-      ]
++
++      {
++        name: '@electron-forge/plugin-webpack',
++        config: { /* ... */ }
++      }
+ ]
+}
+```
+
+### Prefer forge.config.js over package.json config ([#2991](https://github.com/electron-userland/electron-forge/commit/777197e5))
+
+  - This change affects the init and import commands. It changes these commands to create a forge.config.js rather than creating config.forge in your package.json.
+  - The internal signature of `Plugin.getHook(name)` has been changed to `Plugin.getHooks(name)` [#2995](https://github.com/electron/forge/pull/2995) as part of these changes.
+  - This is considered a breaking change for any third-party templates because the base template no longer sets up a config.forge object, and any template-specific mutations to the Forge config should instead be performed on the forge.config.js file.
+  - templates that mutated the Forge config within `package.json` will need to instantiate their own `forge.config.js` or `forge.config.ts`
+
+### Upgraded Maker Wix dependency to `electron-wix-msi@5.0.0` ([3008](https://github.com/electron/forge/pull/3008)))
+  - This upgrade includes a rename from `appIconPath` to `icon` in the config ([#153](https://github.com/electron-userland/electron-wix-msi/pull/153)).
+
+### Upgraded required Node.js to 14 LTS ([#2921](https://github.com/electron/forge/pull/2921))
+
+### Upgraded package dependency to `electron-packager@17` ([#2978](https://github.com/electron-userland/electron-forge/pull/2978))
+
+The upgrade to Electron Packager 17 introduces the shiny new `@electron/osx-sign` package for macOS code signing. It's a rewrite of the old `electron-osx-sign` tool with more sensible defaults.
+
+To migrate, we recommend seeing if the default `packagerConfig.osxSign` options work for you and tweaking the default entitlements to your needs. Otherwise, see the `@electron/osx-sign` [MIGRATION.md](https://github.com/electron/osx-sign/blob/main/MIGRATION.md) doc for a 1:1 conversion from the old config options to the new ones.
+
+### Renamed Electron Rebuild config ([#2963](https://github.com/electron-userland/electron-forge/pull/2963))
+
+For consistency with the `packagerConfig` option for `electron-packager`, the field to configure `@electron/rebuild` has now been shortened to `rebuildConfig`.
+
+```diff
+{
+-  electronRebuildConfig: { /* ... */ }
++  rebuildConfig: { /* ... */ }
+}
+```
+
+### Renamed `identity-validation` to`identityValidation` ([#2959](https://github.com/electron-userland/electron-forge/pull/2959))
+
+The field to configure `identity-validation` has now been shortened to `identityValidation`.
```

Comment:
maybe mention this is for the osxSign part of the config

Reply:
can you elaborate? I'm not sure I follow

Reply:
This property is pretty deep in the Forge config.

```js
module.exports = {
 packagerConfig: {
   osxSign: {
     identityValidation: 'foo'
   }
 }
}
```

It'd be good to make that clearer

Reply:
On second thought, I actually think there are more breaking changes in the `@electron/osx-sign` migration than just this one property, so I think this section is misleading.

We can flesh this out tomorrow but the tool's default code signing entitlements are different (and should work out of the box without additional configuration) and the way we set entitlements is different as well I think.

A hacky 1:1 migration looks something like this https://github.com/electron/osx-sign/blob/main/MIGRATION.md#direct-migration

Reply:
Good catch- I've moved this breaking change to be a subheading within the `electron-packager@17` changes so that it's more logically cohesive with the `osx-sign` changes as a preliminary step. I've also added a title in the diff for `identityValidation` so that it specifies where in the config it lies.

We should do another review to flesh this section out and make sure we catch all breaking changes tomorrow morning though.